services/_shared/personas/walkthrough/dispatch.py

```
@@ -0,0 +1,278 @@
+    extra: dict[str, Any] = {
+        "installation_id": installation_id,
+        "pr": f"{owner}/{repo_name}#{pull_number}",
+        "phase": phase,
+        "kind": type(e).__name__,
+    }
+    if isinstance(e, httpx.HTTPStatusError):
+        extra["status_code"] = e.response.status_code
+    log.warning("walkthrough_fetch_failed", extra=extra)
+
+
+def _emit_degraded_metric(degraded: bool) -> None:
```

Ruff (0.15.20)

[warning] Boolean-typed positional argument in function definition

(FBT001)